### PR TITLE
Initi script create

### PR DESCRIPTION
**Pull request by iruzevic**
Init

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+const checkComposerVersion = async () => exec(`composer -v`);
+/**
+ * Check WP-CLI version
+ */
+const checkWPCliVersion = async () => exec(`wp --info`);
+
+/**
+ * Check Git version
+ */
+const checkGitVersion = async () => exec(`git --version`);
```
Comment (goranalkovic-infinum) [inline src/helpers.js]:
```suggestion
const checkNodeVersion = async () => exec('node -v');
/**
 * Check Composer version
 */
const checkComposerVersion = async () => exec('composer -v');
/**
 * Check WP-CLI version
 */
const checkWPCliVersion = async () => exec('wp --info');

/**
 * Check Git version
 */
const checkGitVersion = async () => exec('git --version');
```

File: src/helpers.js [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+	}
+
+	return join(rootDir, 'wp-content', dirname);
+};
+
+/**
+ * Empty the console.
+ */
+export const clearConsole = () => {
+	process.stdout.write('\x1Bc');
```
Comment (goranalkovic-infinum) [inline src/helpers.js:95]: Can't this just be `console.clear()`?

File: src/helpers.js [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+/**
+ * Checks if the setup already exists.
+ *
+ * @param {string} folderPath Path to the folder.
+ * @param {string} type Type of the setup.
+ */
+export const checkIsSetupExists = (folderPath, type) => {
+	if (existsSync(folderPath)) {
+		console.log('');
+		error(`It looks like your all-ready have a setup ${type} in your project on path ${folderPath}. Please remove it and try again.`);
```
Comment (goranalkovic-infinum) [inline src/helpers.js]:
```suggestion
		error(`It looks like you already have a setup ${type} in your project on path ${folderPath}. Please remove it and try again.`);
```

File: src/helpers.js [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+ * @param {boolean} isFatal Is the step fatal.
+ */
+export const installStep = async ({ describe, thisHappens, isFatal = true }) => {
+	const spinner = ora({ text: describe, color: 'blue' }).start();
+
+	if (!thisHappens) {
+		throw new Error(`Missing 'thisHappens' parameter for step ${describe}, don't know what needs to be done at this step, aborting.`);
+	}
+
+	await thisHappens.then(() => {
```
Comment (goranalkovic-infinum) [inline src/helpers.js]: Either use `await` or `.then()`, but not both 😄

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1 +1,57 @@
...
+Make sure that you have all these packages installed and ready to use on your system. To make sure that they work, run these commands:
+
+- `node -v`
+- `composer -v`
+- `wp --info`
+- `git --version`
+
+If it doesn't return any errors, you are good to go.
+
+## Installation theme
```
Comment (iobrado) [inline README.md]: Maybe `Installation - Theme` would be better. Or `Theme Installation`

File: README.md [hunk 6]
```diff
@@ -1 +1,57 @@
...
+
+Navigate to any folder in your WordPress project where you want to install a new theme and run this command:
+
+`npx eightshift-create theme`
+
+The script will install the latest version of the Setup script and run it. After the script is finished, please follow the instructions provided by the setup script.
+
+## Installation plugin
```
Comment (iobrado) [inline README.md]: Same as ⬆️

**Review by iobrado — DISMISSED** [review]

File: src/helpers.js (hunk 3 above)
Comment (ghost) [reply src/helpers.js]: *you

File: src/helpers.js [hunk 7]
```diff
@@ -0,0 +1,327 @@
...
+
+	if (args.libsRepoBranch) {
+		params += ` --g_libs_version='${args.libsRepoBranch}'`;
+	}
+
+	if (args.frontendLibsRepoBranch) {
+		params += ` --g_frontend_libs_version='${args.frontendLibsRepoBranch}'`;
+	}
+
+	let msg = 'To finish your setup please run one the following commands:\n\n';
```
Comment (piqusy) [inline src/helpers.js]:
```suggestion
	let msg = 'To finish your setup please run one of the following commands:\n\n';
```

**Review by piqusy — DISMISSED** [review]
Nice work 👏🏼 

Left a suggestion

File: src/helpers.js (hunk 2 above)
Comment (iruzevic) [reply src/helpers.js:95]: I have no idea this was taken from the previous version

File: src/helpers.js (hunk 4 above)
Comment (iruzevic) [reply src/helpers.js]: heheh taken from the previous version :D I hope on you to fix this 🗡️

File: src/helpers.js (hunk 2 above)
Comment (goranalkovic-infinum) [reply src/helpers.js:95]: Ok, we'll deal with it later 😄

File: src/helpers.js (hunk 4 above)
Comment (goranalkovic-infinum) [reply src/helpers.js]:
```js
try {
	await thisHappens();
	spinner.succeed();
} catch (exception) {
	console.log(exception);
	spinner.fail();
	error(exception);

	if (isFatal) {
		error(`'${describe}' was a required step, exiting.`);
		process.exit(1);
	}
}
```

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

**Review by piqusy — DISMISSED** [review]

**Review by iobrado — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Review by goranalkovic-infinum — APPROVED** [review]

**Pull request merged**